guides/getting_started.md

```diff
@@ -8,6 +8,18 @@ order: 10
 
 Let's create our first MUD project!
 
+You will need:
+- git ([download](https://git-scm.com/downloads))
+- foundry (forge, anvil, cast) ([download](https://book.getfoundry.sh/getting-started/installation), make sure to `foundryup` at least once)
+- node.js (v16+) ([download](https://nodejs.org/en/download/))
+- yarn (after installing node: `npm install yarn --global`)
+
+!!!
+If you are running Windows, you will need to setup WSL and configure it properly. The libraries and tools listed above need to be installed in the WSL. All commands below also need to be executed in the WSL. 
+
+We recommend reading this guide from Microsoft to get started: [WSL Introduction](https://learn.microsoft.com/en-us/training/modules/wsl/wsl-introduction/introduction)
```

Comment:
As a broad note on linking, it's usually advised to link text in the context of prose rather than the format of "Go here to read a thing: [here]"

So this would look something like:

> We recommend reading the [WSL guide from Microsoft](https://learn.microsoft.com/en-us/training/modules/wsl/wsl-introduction/introduction) to get started.

```suggestion
We recommend reading the [WSL guide from Microsoft](https://learn.microsoft.com/en-us/training/modules/wsl/wsl-introduction/introduction) to get started.
```